pandas/core/series.py

```diff
@@ -2352,16 +2383,26 @@ def combine(self, other, func, fill_value=None):
 
     def combine_first(self, other):
         """
-        Combine Series values, choosing the calling Series's values
-        first. Result index will be the union of the two indexes
+        Combine Series values, choosing the calling Series's values first.
 
         Parameters
         ----------
         other : Series
+            The value(s) to be combined with the `Series`.
 
         Returns
         -------
-        combined : Series
+        Series
+            The result of combining the Series with the other object.
+        
+        See Also
+        --------
+        Series.combine : Perform elementwise operation on two Series
+                         using a given function
```

Comment:
Same as before.

Reply:
looks like VS Code ignored my change there, strange..

Reply:
Did the changes from online editor

Reply:
The right indentation is 4 spaces more than the previous line (in both docstrings)

Reply:
you mean, like this?
```
   Series.combine : Perform elementwise operation on two Series
                                using a given function
```

Reply:
Since the beginning of the line, aligned with the first `i` of `Series.combine`. Same indentation as Python.